src/runtime/server/plugins/refresh-token.server.ts

```diff
@@ -0,0 +1,39 @@
+import { _fetch } from '../../utils/fetch'
```

Comment:
Here I have some points:
- Do we need this plugin? Can we not rely on the server-side execution of the `plugin.ts` we already defined on the app to refresh the token on load?
- The existance of this plugin forces the user to have a server, without it refresh could also work for fully-static apps. So if we keep this plugin (depends on the answer to my previous question) we should at least make it opt-in to ensure that static apps still work